measure_extinction/extdata.py

```diff
@@ -201,9 +198,11 @@ class ExtData:
     waves : dict of key:wavelengths
         key is BAND, IUE, IRS, etc.
 
-    ext : dict of key:E(lambda-v) measurements
+    ext : dict of key:E(lambda-v) or A(lambda)/A(V) measurements
 
-    uncs : dict of key:E(lambda-v) measurement uncertainties
+    uncs : dict of key:E(lambda-v) or A(lambda)/A(V) measurement uncertainties
```

Comment:
Instead of "V",this should be "x" as the relative band is now set by type_rel_band.